components/feature/sitepermissions/src/test/java/mozilla/components/feature/sitepermissions/SitePermissionsFeatureTest.kt

```diff
@@ -285,6 +289,75 @@ class SitePermissionsFeatureTest {
         }
     }
 
+    @Test
+    fun `only autoplay requests should use default rules instead of site rules`() {
+        val siteRules = SitePermissions(
+            location = BLOCKED,
+            camera = BLOCKED,
+            notification = BLOCKED,
+            microphone = BLOCKED,
+            autoplayAudible = BLOCKED,
+            autoplayInaudible = BLOCKED,
+            origin = "any",
+            savedAt = 1
+        )
+        val defaultRules = SitePermissionsRules(
+            location = SitePermissionsRules.Action.ALLOWED,
+            camera = SitePermissionsRules.Action.ALLOWED,
+            notification = SitePermissionsRules.Action.ALLOWED,
+            microphone = SitePermissionsRules.Action.ALLOWED,
+            autoplayAudible = SitePermissionsRules.Action.ALLOWED,
+            autoplayInaudible = SitePermissionsRules.Action.ALLOWED
+        )
+
+        doReturn(siteRules).`when`(mockStorage).findSitePermissionsBy(anyString())
+        sitePermissionFeature.sitePermissionsRules = defaultRules
```

Comment:
👍  Additionally we could also add a case where `sitePermissionsRules` are not provided but `SitePermissions` are